poodle/src/components/Mypage/Mypage.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { FC } from 'react';
+import * as S from '@/styles/Mypage';
+import ContentHeader from '@/components/default/common/ContentHeader'
+import ProgressBar from './ProgressBar/ProgressBar';
+
+enum MyInfoTitle {
+    이름,
+    성별,
+    최종제출,
+    '전형료 납부',
+    '우편물 수령',
+    특기사항
+}
+
+const dummyData = {
+    name: '김해건',
+    gender: '남',
+    finallySubmit: '미완료',
+    payMoney: '납부 전',
+    recieveEmail: '수령 전',
+    specialThing: '검정고시'
+}
+
+const Mypage: React.FC<{}> = () => {
```

Comment:
```suggestion
const Mypage: React.FC = () => {
```
props 가 없으면 제네릭 안넘겨도 됩니다.